lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -142,7 +144,7 @@ def __init__(self,
             self._assigned_ip_blocks = store.AssignedIpBlocksSet(client)
```

Comment:
We should update the redis client (as we're getting the default client), to use the address and port retrieved from the config. Look up mobility_store.py as I think we are also getting the default client on some places there, ideally we should use the one we're creating here. You can update it in a separate PR if you prefer.